crime-application-tracking/src/main/resources/application.yaml

```diff
@@ -18,3 +18,12 @@ spring:
       resource-server:
         jwt:
           issuer-uri: ${JWT_ISSUER_URI}
+
+sentry:
+  dsn: ${SENTRY_DSN}
+  environment: ${SENTRY_ENV}
+  release: 0.0.1
+  servername: aws-eks-container
+  send-default-pii: true
```

Comment:
I assume we're doing this in other places but it did raise a small alarm in my head about what Sentry captures and apparently this can capture things like auth headers, are we happy with that? 

There's maybe a bigger discussion also around Sentry capturing request bodies and whether there's something we need to consider with the kind of data we send as well, but I'll maybe bring that up in standup or our next retro or something. :)

Reply:
Ah good to know, thanks. :) Still might be worth us doing a bit of digging in the future, particularly with the request bodies as I wouldn't expect Sentry to know what to scrub from that, but that's not to do with this pii setting anyway. Happy to approve this PR.